lib/assessment.rb

```diff
@@ -13,28 +14,29 @@ def longest(enum)
   # enumerable supplied (i.e. apply the `+` operator)
   # Assume each entry implements a `+` method
   def sum(enum)
-    # Your code goes here
+    enum.reduce(:+)
```

Comment:
Correct; however, this will only work in Ruby 2.0 and above. For older versions of Ruby, you will need to explicitly call `&` before the `:+` symbol.